audio/common/alsa.h

```diff
@@ -14,43 +14,98 @@
  */
 
 
-#ifndef _RETROARCH_ALSA
-#define _RETROARCH_ALSA
+#ifndef RETROARCH_ALSA_H
+#define RETROARCH_ALSA_H
 
-#include <boolean.h>
-#include <queues/fifo_queue.h>
-#include <rthreads/rthreads.h>
-
-/* Header file for common functions that are used by alsa and alsathread. */
+#include <stdbool.h>
+#include <stddef.h>
+#include <alsa/asoundlib.h>
 
 /**
- * Used for info that's common to all pcm devices
- * that's relevant for our purposes.
+ * @brief Common information for PCM devices.
  */
-typedef struct alsa_stream_info
-{
-   size_t buffer_size;
-   size_t period_size;
-   snd_pcm_uframes_t period_frames;
-   unsigned int frame_bits;
-   bool has_float;
-   bool can_pause;
+typedef struct alsa_stream_info {
+    size_t buffer_size;
+    size_t period_size;
+    snd_pcm_uframes_t period_frames;
+    unsigned int frame_bits;
+    bool has_float;
+    bool can_pause;
```

Comment:
Please follow the [code style](https://docs.libretro.com/development/coding-standards/)